### ISSUE TITLE
rule function-comma-newline-after not working correctly

### ISSUE THREAD

**Issue by evilebottnawi** [stylelint/stylelint#893]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?
About rule:

```
function-comma-newline-after
Require a newline or disallow whitespace after the commas of functions.
```

If i use function with multiple argument, rule not throw error if argument in one line

> Which rule, if any, is this issue related to?

`function-comma-newline-after`

> What CSS is needed to reproduce this issue?

```scss
.test {
    transform:
        translate(
            1, 1
        );
}

.test-1 {
    transform: translate(
        1, 1
    );
}

.test-2 {
    transform:translate(
        1, 1
    );
}
```

> What stylelint configuration is needed to reproduce this issue?

```json
{
  "rules": {
    "function-comma-newline-after": "always-multi-line"
  }
}
```

> Which version of stylelint are you using?

`4.5.1`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

PostCSS plugin

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

SCSS

> What did you expect to happen?

After each commas require a newline, if no newline throw error.

> What actually happened (e.g. what warnings or errors you are getting)?

Nothing

**Comment by davidtheclark** [comment]
So you're saying the rule just doesn't work, right?

**Comment by evilebottnawi** [comment]
@davidtheclark in description rule says `Require a newline or disallow whitespace after the commas of functions.`. But in above examples rule not throw errors.

**Comment by ntwb** [comment]
Would this CSS not be considered a warning:

```css
.test-2 {
    transform:translate(
        1,
        1
    );
}
```

**Comment by evilebottnawi** [comment]
@ntwb i think this be considered a warning

**Comment by davidtheclark** [comment]
Fixed released in v5.
